### PR TITLE
system: add ramspeed test.

### PR DESCRIPTION
**Pull request by FASTSHIFT**
Signed-off-by: pengyiqiang <pengyiqiang@xiaomi.com>

## Summary
Measure memory speed using system and internally implemented `memcpy()` and `memset()`.

## Impact
Add new features.

## Testing
daily test.

### CODE REVIEW

File: system/ramspeed/ramspeed.c [hunk 1]
```diff
@@ -0,0 +1,435 @@
+/****************************************************************************
+ * apps/system/ramspeed/ramspeed.c
```
Comment (pkarashchenko) [inline system/ramspeed/ramspeed.c:2]: Can you please rename to `ramspeed_main.c`?

File: system/ramspeed/ramspeed.c [hunk 2]
```diff
@@ -0,0 +1,435 @@
...
+
+  print_rate("internal memset():\t", total_size, cost_time);
+}
+
+/****************************************************************************
+ * Public Functions
+ ****************************************************************************/
+
+/****************************************************************************
+ * Name: main
```
Comment (pkarashchenko) [inline system/ramspeed/ramspeed.c:419]:
```suggestion
 * Name: ramspeed_main
```

File: system/ramspeed/ramspeed.c (hunk 1 above)
Comment (FASTSHIFT) [reply system/ramspeed/ramspeed.c]: Done

File: system/ramspeed/ramspeed.c (hunk 2 above)
Comment (FASTSHIFT) [reply system/ramspeed/ramspeed.c]: Done

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**